apps/server/src/modules/user-import/controller/dto/import-user.response.ts

```diff
@@ -12,8 +12,10 @@ export class ImportUserResponse {
 		this.lastName = props.lastName;
 		this.roleNames = props.roleNames;
 		this.classNames = props.classNames;
+		this.externalRoleNames = props.externalRoleNames;
 		if (props.match != null) this.match = props.match;
 		if (props.flagged === true) this.flagged = true;
+		if (props.externalRoleNames) this.externalRoleNames = props.externalRoleNames;
```

Comment:
you already do the assignment to the variable on line 15, so this is not needed.